.github/workflows/release.yml

```diff
@@ -0,0 +1,79 @@
+name: Build and release
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+      with:
+        submodules: recursive
+
+    - name: Set up GCC
+      uses: egor-tensin/setup-gcc@v1
+      with:
+        version: 10
+        platform: x64
+
+    - name: Build libopus
+      shell: bash
+      working-directory: ${{github.workspace}}/opus
+      run: |
+        ./autogen.sh
+        ./configure
+        make -j12
+        sudo make install
+
+    - name: Build LuaJIT
+      shell: bash
+      working-directory: ${{github.workspace}}/LuaJIT/src
+      run: sudo make XCFLAGS+=-DLUAJIT_ENABLE_LUA52COMPAT
+
+    - name: Create Build Environment
+      run: cmake -E make_directory ${{github.workspace}}/release
+
+    - name: Configure CMake
+      shell: bash
+      working-directory: ${{github.workspace}}/release
+      run: cmake .. -DCMAKE_BUILD_TYPE=$BUILD_TYPE
+
+    - name: Build
+      working-directory: ${{github.workspace}}/release
+      shell: bash
+      run: cmake --build . --config $BUILD_TYPE
+
+    - name: Test
+      working-directory: ${{github.workspace}}/test
+      shell: bash
+      run: ./test
+
+    - name: Zip Release
+      working-directory: ${{github.workspace}}
+      shell: bash
+      run: zip -j release.zip test/rosaserversatellite test/librosaserver.so test/libluajit.so
```

Comment:
People might not want to download all the files, because in some updates only "librosaserver.so" can change. Upload files separately in a single Release instead of a zip.

Reply:
I added in those files now, I think this should work like this. I wanted to include the zip process in case someone wanted to download all as one, it's been renamed to allfiles.zip. Included in the release should be all 3 files separately and the zip